asm-renamed/pom.xml

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Wouldn't it be cleaner to just depend on asm in a regular way and shade it during the AspectJ packaging process?

Reply:
Good idea. Funny thatt you are asking about it just now, because I was thinking aloud about it just today in https://github.com/kriegaex/org.aspectj/commit/18f3385e23b13b385edcbcc81dbb52623607dd8f for other reasons (creating dependency-reducesd POMs for future direct deployment to Maven Central). See also the e-mail I just sent you a couple of minutes ago, before reading this comment.

At the time I migrated the shading + relocation process for ASM from Ant to Maven, I not want to migrate all affected artifacts from Maven Assembly Plugin to Maven Shade, it would have been too much at the same time. But AFAIK, Maven Assembly cannot relocate. It is certainly an option to migrate to Maven Shade in the future, though.